pytorch/inference/docker/1.11/py3/cu115/Dockerfile.gpu

```diff
@@ -0,0 +1,285 @@
+ && make install \
+ && cd ../.. \
+ && rm -rf aws-sdk-cpp
+
+# Install NCCL
+RUN cd /tmp \
+ && git clone https://github.com/NVIDIA/nccl.git -b v${NCCL_VERSION}-1 \
+ && cd nccl \
+ && make -j64 src.build BUILDDIR=/usr/local \
+ && rm -rf /tmp/nccl
+
+# https://github.com/docker-library/openjdk/issues/261 https://github.com/docker-library/openjdk/pull/263/files
+RUN keytool -importkeystore -srckeystore /etc/ssl/certs/java/cacerts -destkeystore /etc/ssl/certs/java/cacerts.jks -deststoretype JKS -srcstorepass changeit -deststorepass changeit -noprompt; \
+    mv /etc/ssl/certs/java/cacerts.jks /etc/ssl/certs/java/cacerts; \
+    /var/lib/dpkg/info/ca-certificates-java.postinst configure;
+
+RUN wget --quiet https://download.open-mpi.org/release/open-mpi/v4.1/openmpi-${OMPI_VERSION}.tar.gz \
+ && gunzip -c openmpi-${OMPI_VERSION}.tar.gz | tar xf - \
+ && cd openmpi-${OMPI_VERSION} \
+ && ./configure --prefix=/home/.openmpi --with-cuda \
+ && make all install \
+ && cd .. \
+ && rm openmpi-${OMPI_VERSION}.tar.gz \
+ && rm -rf openmpi-${OMPI_VERSION}
+
+ENV PATH="$PATH:/home/.openmpi/bin"
+ENV LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/home/.openmpi/lib/"
+
+# Install OpenSSH. Allow OpenSSH to talk to containers without asking for confirmation
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends \
+    openssh-client \
+    openssh-server \
+ && mkdir -p /var/run/sshd \
+ && cat /etc/ssh/ssh_config | grep -v StrictHostKeyChecking > /etc/ssh/ssh_config.new \
+ && echo "    StrictHostKeyChecking no" >> /etc/ssh/ssh_config.new \
+ && mv /etc/ssh/ssh_config.new /etc/ssh/ssh_configs \
+ && rm -rf /var/lib/apt/lists/* \
+ && apt-get clean
+
+RUN curl -L -o ~/miniconda.sh https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh \
+ && chmod +x ~/miniconda.sh \
+ && ~/miniconda.sh -b -p ${CONDA_PREFIX} \
+ && rm ~/miniconda.sh \
+ && ${CONDA_PREFIX}/bin/conda update conda \
+ && ${CONDA_PREFIX}/bin/conda install -c conda-forge \
+    python=$PYTHON_VERSION \
+ && ${CONDA_PREFIX}/bin/conda install -y \
+    # conda 4.10.0 requires ruamel_yaml to be installed. Currently pinned at latest.
+    ruamel_yaml==0.15.100 \
+ && ${CONDA_PREFIX}/bin/conda install -y -c anaconda \
+    cython \
+    mkl \
+    mkl-include \
+    parso \
+    scipy \
+    typing \
+ && ${CONDA_PREFIX}/bin/conda clean -ya
+
+# Conda installs links for libtinfo.so.6 and libtinfo.so.6.2 both
+# Which causes "/opt/conda/lib/libtinfo.so.6: no version information available" warning
+# Removing link for libtinfo.so.6. This change is needed only for ubuntu 20.04-conda, and can be reverted
+# once conda fixes the issue: https://github.com/conda/conda/issues/9680
+RUN rm -rf /opt/conda/lib/libtinfo.so.6
+
+RUN ${CONDA_PREFIX}/bin/conda config --set ssl_verify False \
+ && pip install --upgrade pip --trusted-host pypi.org --trusted-host files.pythonhosted.org \
+ && ln -s ${CONDA_PREFIX}/bin/pip /usr/local/bin/pip3
+
+RUN conda install -c pytorch \
+    magma-cuda115==2.6.1 \
```

Comment:
Why does this need to be hardcoded to `2.6.1`?

Reply:
I don't think we need. It was previously pinged to be 2.5.2, and recently released a new 2.6.1. We can try to remove the version tag.